src/NuGet.Server/Default.aspx

```diff
@@ -22,14 +22,20 @@
             In the package manager settings, add the following URL to the list of 
             Package Sources:
             <blockquote>
-                <strong><%= Helpers.GetRepositoryUrl(Request.Url, Request.ApplicationPath) %></strong>
+                <strong><%= 
+                // CodeQL [SM02175] False Positive: Url is validated
```

Comment:
I think it would be better to point to relevant documentation. Right now this statement looks like "trust me, it is fine".

Reply:
Added doc 👍